apps/website/src/pages/show-and-tell/index.tsx

```diff
@@ -122,87 +123,95 @@ const ShowAndTellIndexPage: NextPage<ShowAndTellPageProps> = ({
   );
   const registerObserveElement = useIntersectionObserver(onEntryIntersection);
 
-  // When the user attempts to scroll, we want to debounce scroll snapping
-  const scrollDebounceMs = 100;
-  const scrollThresholdPx = 100;
-  const scrollDebounce = useRef<{
-    start: HTMLElement;
-    distance: number;
+  // Scroll an element into the center of the viewport
+  const scrollTo = useCallback((element: HTMLElement) => {
+    // We use a library to do this, because not every browser supports it
+    // And even in Chrome, smooth breaks for the first/last element of things
+    scrollIntoView(element, {
+      behavior: "smooth",
+      block: "center",
+    });
```

Comment:
My only gripe with this now is that this animation-based scrolling has no way to abort if the user starts scrolling again, so you somewhat end up fighting it briefly. I'm not sure that's really that much of a problem though